tools/cli/.eslintrc.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
+	// Use root level ESlint configuration.
+	// JavaScript files inside this folder are meant to be transpiled by Webpack.
+	root: true,
+	extends: [ '../../.eslintrc.js' ],
```

Comment:
What does this do that's different from what the file does without any modifications?

Reply:
🤷 I copied the file contents from plugins/jetpack. Reverted in 20ac74d